### PR TITLE
Content and image updates

### PR DESCRIPTION
**Pull request by cmwilson21**
👋 Sorry again for the delay here. I love this course and learned from just reviewing it! I know it took a lot of time to create. Sending over my updates for your review. Let me know if you have any questions over any of it. I based the content review on our [style guide](https://docs.github.com/en/contributing/writing-for-github-docs/style-guide) (mainly things like active vs passive voice, tone, inline links, etc) and updated the images per our [screenshot guidance](https://docs.github.com/en/contributing/writing-for-github-docs/creating-screenshots) for accessibility. Although I have not yet updated the alt-text but that can be done later. 

Happy to discuss any of these changes async or in zoom 👍 

Once finished here, we tag in engineering and format it for the Skills repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,28 +75,31 @@ Let's get started!
...
 
-**What is CodeQL**: [CodeQL](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning-with-codeql) is a static analysis testing tool created by the team at GitHub.  CodeQL is a deep semantic analysis tool that helps you identify security weaknesses such as SQL injection, cross-site scripting, and code injection issues.  
+**What is CodeQL**: _[CodeQL](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning-with-codeql)_ is a static analysis testing tool created by the team -(should we mention which team?)- at GitHub. CodeQL is a deep semantic analysis tool that helps you identify security weaknesses such as SQL injection, cross-site scripting, and code injection issues.
```
Comment (cmwilson21) [inline README.md]: Wondering if we should mention which team created CodeQL here in this bit

File: README.md [hunk 2]
```diff
@@ -211,46 +207,36 @@ This last step will show you how to create a GitHub Issue to track the work that
...
+1. Click the **Code** tab in your repository.
+2. Select the `server` folder.
+3. Select the `routes.py` file.
+4. Click the **Edit** button to the right.
   
-  ![image](https://github.com/leftrightleft/enable-code-scanning/assets/4910518/a5f7b863-aa06-4302-846b-82af27f1fa84)
+  ![edit-button.png](images/edit-button.png)
   
-5. Select **Commit changes...** from the top right 
+5. Edit line 16 by highlighting the SQL statement and replace it with this text: `"SELECT * FROM books WHERE name LIKE %s", name`.
```
Comment (cmwilson21) [inline README.md:232]: This is 100% my own ignorance in this language, but should there be more information as to why these changes are being made? Or is it assumed those taking this course will know why the `+ "%'"` is removed?

File: README.md (hunk 1 above)
Comment (leftrightleft) [reply README.md]: Good question.  I'm not sure it's important to share the complex history of CodeQL in this document.  Let's change this note to look more like this:
```suggestion
**What is CodeQL**: _[CodeQL](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning-with-codeql)_ is a static analysis testing tool that helps you identify security weaknesses such as SQL injection, cross-site scripting, and code injection issues.
```

File: README.md [hunk 3]
```diff
@@ -112,89 +115,82 @@ In this first step, we'll be learning more about CodeQL and how to use it to sec
...
 
-In this exercise, we'll review the CodeQL findings and understand how to triage the alerts.  We have 5 activities we'll accomplish in this step:
-  1. View the status of a CodeQL scan
-  2. View the CodeQL alerts
-  3. Review an alert
-  4. Dismiss an alert
-  5. Create a GitHub issue to track an alert
+In this exercise, we'll review the CodeQL scan results, triage an alert, and create a GitHub issue to track an alert.
+
+**What is GitHub Actions**: GitHub Actions is the automation and CI/CD platform within GitHub.We use GitHub Actions to orchestrate and execute security scans with code scanning. GitHub Actions is a continuous integration and continuous delivery (CI/CD) platform that allows you to automate your build, test, and deployment pipeline. For more information on GitHub Actions, see "[Understanding GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions)."
```
Comment (leftrightleft) [inline README.md]: Missed a space here :)
```suggestion
**What is GitHub Actions**: GitHub Actions is the automation and CI/CD platform within GitHub. We use GitHub Actions to orchestrate and execute security scans with code scanning. GitHub Actions is a continuous integration and continuous delivery (CI/CD) platform that allows you to automate your build, test, and deployment pipeline. For more information on GitHub Actions, see "[Understanding GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions)."
```

File: README.md (hunk 2 above)
Comment (leftrightleft) [reply README.md:232]: I think we explain this pretty well on line [220](https://github.com/leftrightleft/enable-code-scanning/pull/6/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R220).  I don't think we need to go much more into the details here.

**Comment by leftrightleft** [comment]
@cmwilson21 first off, thank you for getting this reviewed.  It's a lot of work updating all these screenshots, grammar errors, and readability issues.  THANK YOU!

I learned a lot from your changes that I'll be sure to correct for the next skills repo I'll be working on :). Specifically, I need to break my habit of using two spaces after a sentence. It's so deeply engrained in me to mash that spacebar one extra time just for good measure. Also, my screenshots. I didn't realize we had a standard for screenshots. 

I left a few responses and updates.  Looks good!

Thanks again!

**Review by leftrightleft — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cmwilson21) [reply README.md]: Perfect!

File: README.md (hunk 2 above)
Comment (cmwilson21) [reply README.md:232]: Awesome! Thank you for pointing that out 😄

File: README.md (hunk 3 above)
Comment (cmwilson21) [reply README.md]: Great catch! 😆

**Comment by cmwilson21** [comment]
@leftrightleft - I realized that the md files don't render the double space 😆 I get that habit though!

Thank you for the suggestions and for answering my questions! If you have any questions on our style guide or screenshots, I'm happy to chat about it at anytime. We will be updating the Skills repo soon to include its own style guide for contributing new courses, so it will be readily available for future courses. 👍 

Thank you for providing such thorough content from the get go!

**Comment by leftrightleft** [comment]
@cmwilson21 I think this is ready to be merged.  It looks like there's a conflict, though.  Would you mind taking a look at that?  Let me know if I need to take some action.  Thanks!

**Comment by cmwilson21** [comment]
@leftrightleft Think it's ready now!

**Review by leftrightleft — APPROVED** [review]
